src/main/java/io/github/homchom/recode/sys/sidedchat/ChatPredicates.java

```diff
@@ -2,11 +2,13 @@
 
 import com.google.common.collect.Lists;
 import io.github.homchom.recode.mod.config.Config;
+import io.github.homchom.recode.render.*;
 import net.minecraft.ChatFormatting;
 import net.minecraft.network.chat.Component;
 import net.minecraft.network.chat.TextColor;
+import org.w3c.dom.css.RGBColor;
```

Comment:
What is this for?

Reply:
I may have struggled a bit getting the right class to import, that is not in use and can be removed. My Local currently has me working on the incoming report idea I have.